apps/03TicTacToe.js

```diff
@@ -23,30 +24,129 @@ function printBoard() {
 
 function horizontalWin() {
     // Your code here
+    var resultHor = true;
+
+    for (var i=0; (i<board.length);i++){
+
+      // Check for one Horizontal line
+      resultHor = true;
+      for (var j=0;j<board[i].length;j++) {
+        if (board[i][j] !== playerTurn) {
+          resultHor = false; }
+      }
+      if (resultHor) {
+        return true;
+      }
+    }
+    return false;
+
+    // var resultHor0 = (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn);
+    // var resultHor1 = (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn);
+    // var resultHor2 = (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn);
+    //
+    // return ( resultHor0 || resultHor1 || resultHor2 );
 }
 
 function verticalWin() {
     // Your code here
+    var resultVer = true;
+    for (var i=0; (i<board.length);i++){
+      // Check for one vertical line
+      resultVer = true;
+      for (var j=0;j<board[i].length;j++) {
+        if (board[j][i] !== playerTurn) { resultVer = false; }
```

Comment:
Looks familiar, nice. Let's put the bodies of if blocks on a new line from now on.